lib/ui/pointer.dart

```diff
@@ -380,6 +381,23 @@ class PointerData {
   /// The current angle of the pan/zoom in radians, with 0.0 as the initial angle.
   final double rotation;
 
+  // An optional function that allows the framework to respond to the event
+  // that triggered this PointerData instance.
+  final void Function({bool allowPlatformDefault})? _respond;
+
+  /// Function that the framework/app can call to respond to the native event
+  /// that triggered this PointerData.
+  ///
+  /// The parameter [allowPlatformDefault] allows the platform to perform the
+  /// default action associated to the native event when it's set to `true`.
+  /// The default value depends on the embedder, for example in the web it is
+  /// set to `false`, unless explicitly allowed by the framework by passing `true`.
+  void respond({required bool allowPlatformDefault}) {
```

Comment:
Let's document the contract for when this method is called multiple times for the same event. (Or should that be illegal and throw?)

Reply:
Yes, let me document that. The behavior is that once you call it with "true", you can't ever roll it back :) (which I need to fix on the web implementation btw)

Reply:
I think it's equally important to document the default behavior when this method isn't called.

Reply:
Added!